### PR TITLE
chore: revert key -> main change in window delegate listener

### LINKED ISSUE #19124: Can’t input using macOS “Emoji & Symbols” popover
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 6.0.0-beta.12
* **Operating System:**
  * macOS 10.14.5
* **Last Known Working Electron version:**
  * 6.0.0-beta.11

### Expected Behavior

Selecting a character from the *Emoji & Symbols* popover should enter it to the focused input

### Actual Behavior

The popover closes and no character is input

### To Reproduce

1. `$ git clone https://github.com/javan/electron-quick-start.git -b emoji-and-symbols-bug`
1. `$ npm install`
1.  `$ npm start || electron .`
1. Click the *Edit* → *Emoji & Symbols* menu or press <kbd>⌃</kbd><kbd>⌘</kbd><kbd>Space</kbd> to open the popover
1. Select any character from the popover

### Screenshots

<details><summary>v6.0.0-beta.12 👎</summary>

electron-6-beta-12

</details>

<details><summary>v6.0.0-beta.11 👍</summary>

electron-6-beta-11

</details>

### Additional Information

https://github.com/electron/electron/pull/18995 may be the culprit since it was introduce in beta.12 (cc: @erickzhao  @codebytere)

### PR DESCRIPTION
**Pull request by erickzhao**
#### Description of Change
Fixes #19124 

Reverts #18995, which caused a regression (#19124). According to our discussion in https://github.com/electron/electron/pull/19128#issuecomment-509439237, it would be best to revert the fix for now, especially since this regression has already been backported into 4.2.X, 5.0.X, and 6.0.X.

For now, the test that I wrote in #18995 has also been commented out.

cc @codebytere @zcbenz @nornagon 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Reverted change to focus behavior that broke Character Viewer support on macOS.

### CODE REVIEW

File: spec/api-browser-window-spec.js [hunk 1]
```diff
@@ -1304,33 +1304,36 @@ describe('BrowserWindow module', () => {
...
-        w.focus() // this should not trigger the above listener
-        child.close()
-      })
-
-      // act
-      child.loadURL(server.url)
-      w.show()
-    })
-  })
+  // temporarily removing test due to regression
```
Comment (codebytere) [inline spec/api-browser-window-spec.js]: i'd rather just delete this for now; this is what version control is for :)

Comment (erickzhao) [reply spec/api-browser-window-spec.js]: Will do!

**Pull request merged**